### ISSUE TITLE
[Single-Entry] Test card supports RSV devices

### ISSUE THREAD

**Issue by jayna-SkylightDigital** [CDCgov/prime-simplereport#6692]
## Why should we do this?

- Support for RSV must be added to the frontend single-entry user-facing experience of SimpleReport, to include RSV as part of Multiplex devices that test for RSV, as well as when an RSV-only device has been selected.

## Action Requested
- Add support for RSV to the single-entry frontend user experience.
- Work is at most a unit test?

## Acceptance Criteria
- A SimpleReport user can enter and submit RSV test results using the single-entry user flow.
- AOE should be the same as COVID/flu multiplex 

**Design**
-   [x] Design is ready for handoff (i.e. dev ready)
-   [x] Design includes latest content updates
-   [ ] Capture questions that come up during the content and design work in the comments section

**Review**
-   [ ] Reviewed with design and/or design lead
-   [ ] Reviewed with content strategist for final approval
-   [ ] Reviewed with product and engineering

## Additional Context
- Work is minimal, RSV is a disease type on multiplex 
- Doesn't need org-level feature flag
- Wait until Mike's work is done to point this
- Figma [[URL](https://www.figma.com/file/LglkDsGZDKMZm3XCkJPhfR/SimpleReport-2023%2B?type=design&node-id=3167%3A21805&mode=design&t=5yga8ovMFn6SyFdb-1)]

**Comment by mehansen** [comment]
Work for this ticket:
- blocked by test card refactor
- test that if singleEntryRsvEnabled is true and testCardRefactorEnabled is true that someone can see RSV in the test card and submit a result and it will be sent to RS
- make any changes to make the above happen

**Comment by mpbrown** [comment]
Confirmed that the test card supports RSV with this example on dev3 @mehansen 


### RSV on the test card itself, along with the default COVID AOE
![Image](https://github.com/CDCgov/prime-simplereport/assets/23287037/39a9dbc6-7058-47eb-b716-d40f45e38291)

### RSV appears on results page 
#### (note that a few parts of this page like Print details, View details, and the condition filter don't show RSV yet so that'll likely need another ticket)
![Image](https://github.com/CDCgov/prime-simplereport/assets/23287037/7e70fb9d-018c-4da8-8885-d01e5de259ec)

### RSV result on the test event in dev3 metabase
![Image](https://github.com/CDCgov/prime-simplereport/assets/23287037/9d45caf2-f991-4ad3-a855-8bb2c536532f)


### Test event id on Azure confirms it sends to the FHIR queue/universal pipeline correctly
![Image](https://github.com/CDCgov/prime-simplereport/assets/23287037/8de12b10-1189-40f6-a210-47265a4e0586)

**Comment by mehansen** [comment]
closing as completed since @mpbrown's  work on the test card refactor has already satisfied these acceptance criteria
